src/views/Team.vue

```diff
@@ -0,0 +1,169 @@
+<template>
+  <div class="container">
+    <div v-for="(team, idx) in fantasyTeams" :key="idx">
+      <div class="row align-items-center profile-header">
+        <div class="col-md text-center">
+          <h2>{{ team.name }}</h2>
+          League Ranking: {{ team.rank }}<br />
+          Total Score: {{ team.score }}<br />
+          Today's Score: {{ team.todaysScore }}<br />
+        </div>
+        <b-table
+          striped
+          :items="computedData"
+          :fields="computedFields"
+          :sort-by.sync="sortBy"
+          :tbody-tr-class="rowClass"
+        >
+        </b-table>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import { QUERY_TEAM } from "../constants/graphQLqueries/graphQLqueries";
+import { QUERY_SCORING_TYPES_FOR_TEAM } from "../constants/graphQLqueries/graphQLqueries";
+
+var getScoringData = function(scoring, id) {
```

Comment:
Function for getting loop through sub variables to get the exact scoring data in an array